services/core-api/app/api/now_applications/resources/now_application_delay_resource.py

```diff
@@ -67,6 +77,11 @@ def put(self, now_application_guid, now_application_delay_guid):
         if not now_app:
```

Comment:
This check should be moved up a bit?

Reply:
Can you elaborate? It is at the very top,  directly under where now_app is defined. Github is showing it weird.

Reply:
This is the PUT check, not the POST.